src/utils/getAssetsUrl.test.ts

```diff
@@ -1,10 +1,36 @@
 import { ASSETS_URL } from './constants';
 import { getAssetsUrl } from './getAssetsUrl';
 
+const uploadCareImage = {
+    version: 2,
+    uuid: '43258238-9e71-49fa-a35e-dca5baf16a3b',
+    filename: 'prezly-photographe.png',
+    mime_type: 'image/png',
+    size: 74234,
+    original_width: 1162,
+    original_height: 369,
+    effects: ['/crop/369x369/0,0/', '/preview/'],
```

Comment:
Not sure if `preview` effect is present on real images, but for our use-case it's better to remove it when applying effects to the URL. Can you update the logic to filter out, as well as cover that in the tests?